vita3k/modules/SceCtrl/SceCtrl.cpp

```diff
@@ -20,12 +20,28 @@
 #include <ctrl/ctrl.h>
 #include <ctrl/functions.h>
 #include <kernel/types.h>
-
+#include <modules/tracy.h>
 #include <util/log.h>
 
 #include <algorithm>
 #include <array>
 
+#ifdef TRACY_ENABLE
+const std::string tracy_module_name = "SceCtrl";
+#endif // TRACY_ENABLE
+
+template <>
+std::string to_debug_str<SceCtrlPadInputMode>(const MemState &mem, SceCtrlPadInputMode mode) {
+    switch (mode) {
+    case SCE_CTRL_MODE_DIGITAL:
+        return "SCE_CTRL_MODE_DIGITAL";
+    case SCE_CTRL_MODE_ANALOG:
+        return "SCE_CTRL_MODE_ANALOG";
+    case SCE_CTRL_MODE_ANALOG_WIDE:
+        return "SCE_CTRL_MODE_ANALOG_WIDE";
+    }
```

Comment:
Maybe add something for other values. I'm going to use 
`default: return to_debug_str(mem, static_cast<typename std::underlying_type<SceCtrlPadInputMode>::type>(mode));`
or something like